app/artifact-cas/internal/server/http.go

```diff
@@ -43,8 +51,40 @@ func NewHTTPServer(c *conf.Server, logger log.Logger) *http.Server {
 	if c.Http.Timeout != nil {
 		opts = append(opts, http.Timeout(c.Http.Timeout.AsDuration()))
 	}
+	// Load the key on initialization instead of on every request
+	// TODO: implement jwks endpoint
+	rawKey, err := os.ReadFile(authConf.RobotAccountPublicKeyPath)
```

Comment:
Please add the comment here about the key pair. This is not meant to be a robot account public key. We use the Robot Account Public Key here for json token verification.